packages/bz2.rb

```diff
@@ -49,6 +36,10 @@ def self.install
     system "ln -s libbz2.so.1.0.6 #{CREW_DEST_LIB_PREFIX}/libbz2.so.1.0"
     system "ln -s libbz2.so.1.0.6 #{CREW_DEST_LIB_PREFIX}/libbz2.so.1"
     system "ln -s libbz2.so.1.0.6 #{CREW_DEST_LIB_PREFIX}/libbz2.so"
+
+    # Move manpages
+    system "mkdir -p #{CREW_DEST_PREFIX}/share"
```

Comment:
@JL2210: Why didn't suggest `FileUtils.mkdir_p "#{CREW_DEST_PREFIX}/share"` here also?

Reply:
Forgot about it. Sorry.